geode-core/src/main/java/org/apache/geode/distributed/internal/LocatorLoadSnapshot.java

```diff
@@ -440,6 +460,24 @@ private void removeFromMap(Map<String, Map<ServerLocation, LoadHolder>> map, Str
     groupMap.remove(location);
   }
 
+  private void removeFromMap(Map<String, Map<ServerLocationAndMemberId, LoadHolder>> map,
```

Comment:
This is a new method and should be, at leat, unit tested.
You can make it package private, annotate it with `@VisibleForTesting` and access it directly from `LocatorLoadSnapshotJUnitTest` and/or `LocatorLoadSnapshotIntegrationTest`.

Reply:
done